src/components/SubscribeForm.js

```diff
@@ -161,6 +160,7 @@ class SubscribeForm extends Component {
   render() {
     return (
       <SForm
+        className={this.props.className}
```

Comment:
@mikedemarais why are you passing a classname?

Reply:
@pedrouid so that it can receieve styles that we may pass to it from parent components. 

like i did here (fingers crossed this link works im on mobile) https://github.com/balance-io/balance-manager/pull/243/files#diff-84099094c0c18f11b3c43fe0e8959b97R129

Reply:
Strange that it wouldn't work otherwise because you can see that it works throughout the app. 

The only scenarios where I've found this, was for Pure Components where you can see that I usually spread the props to the highest component

Example
```
const MetamaskLogo = ({ size, ...props }) => (
  <StyledMetamaskLogo size={size} {...props}>
    <img src={metamaskOriginal} alt="metamask" />
  </StyledMetamaskLogo>
);
```